quantus/functions/perturb_func.py

```diff
@@ -556,3 +555,31 @@ def no_perturbation(arr: np.array, **kwargs) -> np.array:
          Array unperturbed.
     """
     return arr
+
+
+def random_uniform_noise(arr: np.ndarray, amplitude=1e-6, **kwargs) -> np.ndarray:
+    """
+    Parameters
+    ----------
+    arr: np.ndarray
+         3D tensor representing image
+    amplitude: float
+                 Upper bound for noise generated, default 1e-6
+    kwargs:
+        Keyword arguments, unused
+
+    Returns
+    -------
+    arr: np.ndarray
+         Array of the same shape as input
+    """
+    arr_shape = arr.shape
+    if len(arr_shape) == 4:
+        noise_shape = arr.shape[1:]
+    elif len(arr_shape) == 3:
+        noise_shape = arr_shape
+    else:
+        raise ValueError("random_noise supports only 3D and 4D")
+
+    noise = np.random.uniform(size=noise_shape, low=-1 * amplitude, high=amplitude)
```

Comment:
why not `low=-amplitude`?

Reply:
Honestly, I don't know. Changed to `low=-amplitude`